src/agents/ipp_group.jl

```diff
@@ -5106,11 +5165,14 @@ function solve_agent_problem_ipp_cap(
             d in model_data.index_d,
             t in model_data.index_t,
         ],
+        # ipp.rho_E_my(p, k, z, d, t) * (
+        #     ipp.x_E_my(p, z, k) - sum(
+        #         ipp.x_R_my(Symbol(Int(y_symbol)), p, k, z) for y_symbol in
+        #         model_data.year(first(model_data.index_y)):model_data.year(y)
+        #     ) - ipp.x_R_cumu(p, k, z)
+        # ) - y_E_bounds[y, p, k, z, d, t] >= 0
```

Comment:
Should this be deleted?

Reply:
Yeah, we can delete them so the code looks better. I'm just used to keep things in comment in case we need them in the future (I know, not a good practice).